app/views/content/train-to-be-a-teacher/if-you-have-a-degree/_events-near-you.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="events-near-you">
+  <div class="events-near-you__next-event">
+    <div>
+      <h2>
```

Comment:
@peteryates One for build rather than prototype (which I'm sure you'll know already!) - there are too many h2s in this section - VoiceOver reads out four separate h2s